budget/app.py

```diff
@@ -106,8 +137,93 @@ def compare_budgets_and_users(account_id, users):
 
 def create_budget(synapse_id, team):
   '''Creates an AWS budget for a synapse user id'''
-  # TODO implement
-  log.info('Placeholder -- this will make a budget')
+  log.debug(f'Creating budget for synapse user {synapse_id}, member of team {team}')
+
+  team_budget_rules = TEAM_BUDGET_RULES.get('teams').get(team)
+  if not team_budget_rules:
+    raise ValueError(f'No budget rules available for team {team}')
+  budget_amount = team_budget_rules['amount']
+  budget_period = team_budget_rules['period']
+  budget = {
+    'BudgetName': f'{BUDGET_NAME_PREFIX}{synapse_id}',
+    'BudgetLimit': {
+      'Amount': budget_amount,
```

Comment:
No, this is the budget per user.

Reply:
I think this will be significant in documentation... If the total budget for a project is set, the community manager will need to slice it into per-user allocations in advance. In the case that the number of users in the project increases, we will probably not want to revise downwards the budgets for all other users.

Reply:
The current MVP work doesn't take into account changes to budgets for teams, e.g. if they decide to raise the budget from 2000 per user to 2500 per user. I'll file an issue for that.